src/js/spotify.js

```diff
@@ -185,4 +185,40 @@ export function followShow(showId) {
 export function followAlbum(albumId) {
     return makeRequest(`https://api.spotify.com/v1/me/albums?ids=${albumId}`, getRequestHeadersWithToken('PUT'))
         .then(response => response).catch(e => alert("Error: can't follow album" + e));
+}
+
+/**
+ * Remove user saved show
+ * @param  {} showId - Show Id.
```

Comment:
в фигурных скобках нужно указать тип
```suggestion
 * @param  {integer} showId - Show Id.
```